src/uint/boxed/encoding/serde.rs

```diff
@@ -0,0 +1,78 @@
+//! Support for serdect on [`BoxedUint`]
+
+use serdect::serde::{de, Deserialize, Deserializer, Serialize, Serializer};
+
+use crate::BoxedUint;
+
+impl<'de> Deserialize<'de> for BoxedUint {
+    fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
+    where
+        D: Deserializer<'de>,
+    {
+        let mut buffer = Self::zero().to_le_bytes();
+        serdect::array::deserialize_hex_or_bin(buffer.as_mut(), deserializer)?;
```

Comment:
Doesn't work because it doesn't know the size and one() obviously isn't good where as_mut overflows silently.

Reply:
You need to use `serdect::slice` if the size isn't known a priori

Reply:
hmm I thought I tried (the alloc variant) that and it gave some weird runtime behaviour